packages/table/src/table.tsx

```diff
@@ -785,7 +985,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         }
     }
 
-    private renderColumnHeader() {
+    private renderColumnHeader(columnIndexStart?: number, columnIndexEnd?: number) {
```

Comment:
Refactor to one parameter and rename to `numFrozenColumns?: number`. If that parameter is defined, then set `columnIndexStart = 0` and `columnIndexEnd = numFrozenColumns - 1`.